blockchain/scripts/deploy.ts

```diff
@@ -2,25 +2,28 @@ import { ethers } from "hardhat";
 import "dotenv/config";
 import * as nftJson from "../artifacts/contracts/DungeonDefenders.sol/DungeonDefenders.json";
 import * as gemJson from "../artifacts/contracts/Gems.sol/Gems.json";
+import * as stakingJson from "../artifacts/contracts/StakingNFT.sol/StakingContract.json";
 // import { connectToWallet } from "./utils";
 
 async function main() {
   // const { signer } = await connectToWallet();
   const [signer] = await ethers.getSigners();
 
+  // Deploy Gems
   console.log("Deploying GEMS contract");
   const TokenFactory = new ethers.ContractFactory(
     gemJson.abi,
     gemJson.bytecode,
     signer
   );
-  const tokenContract = await TokenFactory.deploy();
+  const totalSupply = ethers.utils.parseEther("100000000");
```

Comment:
Is this a capped total supply?

Reply:
Ill have to check on that, I think owner can still mint
I took an arbitrary number just to test it out, nothing definitive on that !